tools/downloader/list_topologies.yml

```diff
@@ -13,11 +13,9 @@
 # limitations under the License.
 
 topologies:
-#
-# PUBLIC TOPOLOGIES
-#
   - name: "densenet-121"
     description: "DenseNet-121 (https://github.com/shicai/DenseNet-Caffe). The following Caffe patch must be used to be able to convert this model: https://github.com/BVLC/caffe/pull/3057/files"
+    type: "classification"
```

Comment:
Could you rename the field to `task_type`? I think that would be a better description of it.

Reply:
I copied it from our internal repo, but this is better field name here. Renamed